From 887446d4aa3e038771336e968fd03946f284c328 Mon Sep 17 00:00:00 2001
From: SyncingShip
Date: Thu, 17 Oct 2019 23:23:05 +0100
Subject: [PATCH] ShowRunDepth

---
 UIScripts.lua | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/UIScripts.lua b/UIScripts.lua
index ba1c1b4..35de937 100644
--- a/UIScripts.lua
+++ b/UIScripts.lua
@@ -114,6 +114,12 @@ function ShowCombatUI( flag )
 		return
 	end
 
+	if ScreenAnchors.RunDepthId == nil then
+		ShowDepthCounter()
+	else
+		HideDepthCounter()
+	end
+	
 	if not ConfigOptionCache.ShowUIAnimations then
 		return
 	end
-- 
2.16.1.windows.4
